codespell_lib/data/dictionary.txt

```diff
@@ -8246,6 +8246,9 @@ suprize->surprise
 suprized->surprised
 suprizing->surprising
 suprizingly->surprisingly
+suptitle->subtitle
```

Comment:
`suptitle` though rare does exist in some toolboxes (e.g., `matplotlib`)

Reply:
I am aware of that. I'm making a PR for matplotlib actually to change it to subtitle()

Reply:
... and I think this is intentional, because "suptitle" is presumably short for "super-title" (over-title) as opposed to the more common "sub-title" (under-title)

Reply:
yea..i guess that makes sense. I will retract PR.